### PR TITLE
Sica, Falx, Romphaia

### PR DESCRIPTION
**Pull request by indoptogopt**
The article's structure is a bit all over the place honestly... I'm open to suggestions.

### CODE REVIEW

File: gui/encyclopedia/articles/ancient_world/warfare/sica_falx_romphaia.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
...
..., longer, two-handed one. And that the falx made up the middle ground by being wielded sometimes in one hand only, other times again in two hands. In addition, the romphaia is generally associated with the Southern Thracian tribes, in contrast to the falx with the Northern ones like the Getae and Dacians. They all were of great cultural importance and likely even bore some ritual or religious meaning to their owners.\n\nWhat made the swords' design so special was the blade's unusual forward curve, which resulted in a quite unconventional fighting style, including both slashing and thrusting movements.\n\nWielding the swords in both hands on the one hand eliminated the possibility of holding a shield for protection  and only gave comparably short reach for a two-handed weapon, but on the other hand enabled even more powerful blows. So powerful, in fact, the falx and romphaia were alledgedly able to pierce helmets and split shields.\n\nMost famously, the Romans encountered the weapon during the Dacian Wars - and were terrified. Their legions were forced to make multiple modifications to their equipment in mid-campaign for it to better withstand the sheer power of the Dacian strikes."
```
Comment (TheShadowOfHassen) [inline gui/encyclopedia/articles/ancient_world/warfare/sica_falx_romphaia.json]:
```suggestion
  "content": "The sica, the falx, and the romphaia were special single-edged swords wielded by the Thracian people (inhabiting the lands of modern-day Bulgaria and Romania). Now, there is some historical ambiguity and confusion when it comes to the difference between the three kinds. Archeologists have uncovered specimens of very different design and shape. But scholars tend to agree that the sica was the short, one-handed variant and the romphaia the less curved, longer, two-handed one. And that the falx made up the middle ground by being wielded sometimes in one hand only, other times again in two hands. In addition, the romphaia is generally associated with the Southern Thracian tribes, in contrast to the falx with the Northern ones like the Getae and Dacians. They all were of great cultural importance and likely even bore some ritual or religious meaning to their owners.\n\nWhat made the swords' design, so special was the blade's unusual forward curve, which resulted in a quite unconventional fighting style, including both slashing and thrusting movements.\n\nWielding the swords in both hands on the one hand eliminated the possibility of holding a shield for protection  and only gave comparably short reach for a two-handed weapon, but on the other hand enabled even more powerful blows. So powerful, in fact, the falx and romphaia were alledgedly able to pierce helmets and split shields.\n\nMost famously, the Romans encountered the weapon during the Dacian Wars - and were terrified. Their legions were forced to make multiple modifications to their equipment in mid-campaign for it to better withstand the sheer power of the Dacian strikes."
```

**Review by TheShadowOfHassen — COMMENTED** [review]
The comment has a sentence reworded, but other than that it looks good.

**Comment by indoptogopt** [comment]
Ready to be merged?

**Comment by TheShadowOfHassen** [comment]
Yep

**Pull request merged**